components/modals/TsAndCs.tsx

```diff
@@ -0,0 +1,142 @@
+            turpis egestas integer eget. Felis donec et odio pellentesque diam
+            volutpat commodo. Vitae suscipit tellus mauris a diam maecenas.
+            Fames ac turpis egestas maecenas pharetra convallis posuere morbi.
+            Libero nunc consequat interdum varius sit amet mattis. Eget lorem
+            dolor sed viverra ipsum. Tempus quam pellentesque nec nam aliquam
+            sem. Gravida rutrum quisque non tellus. Vivamus at augue eget arcu
+            dictum. Facilisis gravida neque convallis a cras semper auctor neque
+            vitae. Est pellentesque elit ullamcorper dignissim cras tincidunt
+            lobortis feugiat vivamus. Ut enim blandit volutpat maecenas
+            volutpat. Diam vulputate ut pharetra sit amet aliquam id. Metus
+            aliquam eleifend mi in nulla. Vitae aliquet nec ullamcorper sit amet
+            risus nullam. Ipsum suspendisse ultrices gravida dictum fusce.
+            Tellus mauris a diam maecenas sed enim ut sem viverra.
+          </Text>
+          <Text>
+            Tincidunt nunc pulvinar sapien et ligula. Pretium aenean pharetra
+            magna ac placerat. Diam donec adipiscing tristique risus nec feugiat
+            in fermentum posuere. Faucibus interdum posuere lorem ipsum dolor.
+            Ipsum consequat nisl vel pretium lectus quam. Lobortis feugiat
+            vivamus at augue. Sem fringilla ut morbi tincidunt augue interdum
+            velit euismod in. Justo donec enim diam vulputate ut. Bibendum at
+            varius vel pharetra. Vitae congue mauris rhoncus aenean. Accumsan in
+            nisl nisi scelerisque eu. Massa eget egestas purus viverra.
+          </Text>
+          <Text>
+            Sed odio morbi quis commodo. Enim eu turpis egestas pretium aenean
+            pharetra magna ac placerat. Phasellus faucibus scelerisque eleifend
+            donec pretium. Lorem ipsum dolor sit amet consectetur adipiscing
+            elit ut aliquam. Ornare quam viverra orci sagittis eu volutpat.
+            Lectus mauris ultrices eros in cursus turpis massa tincidunt. Quis
+            vel eros donec ac odio tempor. Montes nascetur ridiculus mus mauris
+            vitae ultricies leo. Ullamcorper eget nulla facilisi etiam dignissim
+            diam quis enim lobortis. At augue eget arcu dictum varius. Phasellus
+            faucibus scelerisque eleifend donec pretium vulputate sapien. Sed
+            velit dignissim sodales ut eu sem. Mattis vulputate enim nulla
+            aliquet porttitor lacus. Ultrices dui sapien eget mi proin sed
+            libero.
+          </Text>
+          <Flex w={"100%"} justifyContent={"center"}>
+            <Button
+              onClick={props.onClose}
+              variant={"solid"}
+              bg={"purple"}
+              color={"white"}
+              _hover={{
+                backgroundColor: "purple.300",
+                // color: "black",
```

Comment:
I guess if not needed we can remove

Reply:
🤦 I really need to stop trying to work fast haha
will remove, thanks!